Reach/Payment/Model/ResourceModel/Carrier/CsvHsCode/CSV/RowParser.php

```diff
@@ -82,8 +85,25 @@ private function getHsCode(array $rowData, ColumnResolver $columnResolver)
     {
         $hsCode = $columnResolver->getColumnValue(ColumnResolver::COLUMN_HSCODE, $rowData);
         if ($hsCode === '') {
-            $hsCode = '*';
+            $hsCode = null;
```

Comment:
why was  this change made? It's not super apparent from the other changes here so I wanted to check

Reply:
This change was made because when there was no hs code then it inserted an asterisk into the database.  This caused my check for a value to be true.  so if it was populated with an asterisk it wouldn't go to the default hs code of the merchant that I added in earlier pull requests